pm/sendermonitor.go

```diff
@@ -286,3 +270,76 @@ func (sm *senderMonitor) cleanup() {
 		}
 	}
 }
+
+func (sm *senderMonitor) redeemWinningTicket(ticket *SignedTicket) (err error) {
+	maxFloat, e := sm.MaxFloat(ticket.Ticket.Sender)
+	if e != nil {
+		err = e
+		return
+	}
+
+	// if max float is zero, there is no claimable reserve left or reserve is 0
+	if maxFloat.Cmp(big.NewInt(0)) <= 0 {
+		sm.QueueTicket(ticket)
+		err = errors.Errorf("max float is zero")
+		return
+	}
+
+	// If max float is insufficient to cover the ticket face value, queue
+	// the ticket to be retried later
+	if maxFloat.Cmp(ticket.Ticket.FaceValue) < 0 {
+		sm.QueueTicket(ticket)
```

Comment:
Error needs to be handled

Reply:
included in 9e75eecf but can patch them out separately if you'd like